index.js

```diff
@@ -1 +1,185 @@
-module.exports = {};
+const fs = require('fs-extra');
```

Comment:
I don't see the value `fs-extra` brings over the regular `fs` in this file.

`fs` has a promise API.

Reply:
Done